options/locale/locale_en-US.ini

```diff
@@ -697,6 +697,7 @@ archive.issue.nocomment = This repo is archived. You cannot comment on issues.
 archive.pull.nocomment = This repo is archived. You cannot comment on pull requests.
 
 form.reach_limit_of_creation = You have already reached your limit of %d repositories.
```

Comment:
I think we need use ``TrN`` in here to handle singular and plural  differences.

Reply:
What does "TrN" mean?

Reply:
example:
https://github.com/go-gitea/gitea/blob/eb60a5d05478f56bf15220f49ed78f0f72af5698/templates/repo/activity.tmpl#L44

Reply:
Thanks. Looks like this solution is better, I will try to implement it.